<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRasterMinMaxWidgetBase</class>
 <widget class="QWidget" name="QgsRasterMinMaxWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mLoadMinMaxValuesGroupBox">
     <property name="title">
      <string>Min / Max Value Settings</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="collapsed" stdset="0">
      <bool>true</bool>
     </property>
     <property name="saveCollapsedState" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0">
        <item row="1" column="0">
         <widget class="QRadioButton" name="mCumulativeCutRadioButton">
          <property name="text">
           <string>Cumula&amp;tive
count cut</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">mMinMaxMethodRadioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="mAccuracyLabel">
          <property name="text">
           <string>Accuracy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="mUserDefinedRadioButton">
          <property name="text">
           <string>Use&amp;r defined</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">mMinMaxMethodRadioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="mMinMaxRadioButton">
          <property name="text">
           <string>&amp;Min / max</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">mMinMaxMethodRadioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QgsDoubleSpinBox" name="mCumulativeCutUpperDoubleSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QgsDoubleSpinBox" name="mCumulativeCutLowerDoubleSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="mStatisticsExtentLabel">
          <property name="text">
           <string>Statistics extent</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QgsDoubleSpinBox" name="mStdDevSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QRadioButton" name="mStdDevRadioButton">
          <property name="text">
           <string>Mean +/-
standard de&amp;viation Ã—</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">mMinMaxMethodRadioButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="4" column="1" colspan="3">
         <widget class="QComboBox" name="mStatisticsExtentCombo"/>
        </item>
        <item row="5" column="1" colspan="3">
         <widget class="QComboBox" name="cboAccuracy"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mUserDefinedRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="mMinMaxMethodRadioButtonGroup"/>
 </buttongroups>
</ui>
